keyboards/handwired/illera16/rules.mk

```diff
@@ -0,0 +1,5 @@
+# This file intentionally left blank
+
+RGBLIGHT_ENABLE = yes
+ENCODER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```

Comment:
please move this setting to, new file, `keyboards/handwired/illera16/keymaps/default/rules.mk`